lang/en.yml

```diff
@@ -164,10 +202,10 @@ en:
     REDIRECTTOPAGE: 'A page on your website'
     SINGULARNAME: 'Redirector Page'
     YOURPAGE: 'Page on your website'
-    db_ExternalURL: 'Other website URL'
-    db_RedirectionType: 'Redirect to'
-    has_one_LinkTo: 'Page on your website'
-    has_one_LinkToFile: 'File'
+    db_ExternalURL: 'External URL'
```

Comment:
Why are these changing?

Reply:
https://github.com/silverstripe/silverstripe-cms/pull/3080#discussion_r2094630679

Reply:
`db_ExternalURL` is changing because in CMS 5 it has `new TextField("ExternalURL", _t(__CLASS__.'.OTHERURL', "Other website URL"))` in `RedirectorPage::getCMSFields()` i.e. it's the default value in PHP

In CMS 6 we removed that line, presumably due to improvements in scaffolding

I'll assume all the other changes in this file are for similar reasons

What's kind of annoying is that transifex has no concept of CMS majors, we pretty much have to pick one here which should be CMS 6, so I think it's best to leave things as they are in the PR

Reply:
> we pretty much have to pick one here which should be CMS 6

Why should it be the CMS 6 one though? I didn't notice that line in CMS 5 - I was just looking for `provideI18nEntities` and `fieldLabels` which are where these should ideally be set.

Probably when we removed that line, we didn't realise there would be this change as a result. If that's the case, it would mean that this is an unintended change that we should revert.

If you agree with that assessment I'll add the relevant code back into PHP-land to revert this unexpected change.
If you think we should stick with this PR I'm happy for that too, let me know either way.

I'll hold off updating behat tests pending your decision.

Reply:
I think we just need to let CMS 5 break here to some degree here (and it's a very small degree) otherwise we're going to have this basically redundant code sitting around in CMS 6 purely to retain the CMS 5 era source translation values, which we'll then need to retain in CMS 7 and beyond.

I guess this is just one of the unintended side-effects of having automatically generated translation keys